>>> MyList = __import__('1-my_list').MyList
>>> my_list = MyList()

>>> print(my_list)
[]

>>> my_list.print_sorted()
[]

>>> isinstance(my_list, list)
True

>>> my_list.append(1)
>>> my_list.append(4)
>>> my_list.append(2)
>>> my_list.append(3)
>>> my_list.append(5)

>>> print(my_list)
[1, 4, 2, 3, 5]

>>> my_list.print_sorted()
[1, 2, 3, 4, 5]

>>> print(my_list)
[1, 4, 2, 3, 5]

>>> my_list.clear()
>>> print(my_list)
[]

>>> my_list.append(3)
>>> my_list.append(3)
>>> my_list.append(3)
>>> my_list.append(3)

>>> print(my_list)
[3, 3, 3, 3]

>>> my_list.print_sorted()
[3, 3, 3, 3]

>>> print(my_list)
[3, 3, 3, 3]

>>> len(my_list)
4

>>> my_list.append(2)
>>> my_list[-1]
2

>>> my_list.print_sorted()
[2, 3, 3, 3, 3]

>>> my_list.pop(0)
3

>>> print(my_list)
[3, 3, 3, 2]

>>> print(my_list[4])
Traceback (most recent call last):
            ...
IndexError: list index out of range

>>> my_list2 = MyList([1, 2, 3, 4])
>>> type(MyList.__dict__['print_sorted'])
<class 'function'>

>>> print(my_list2)
[1, 2, 3, 4]

>>> my_list2.print_sorted()
[1, 2, 3, 4]

>>> my_list2 = MyList([-5, -2, -4, 0])
>>> my_list2.print_sorted()
[-5, -4, -2, 0]

>>> my_list2 = MyList([1, 2, 3, 4])

>>> my_list2 = MyList([4, 2], [1, 3])
Traceback (most recent call last):
            ...
TypeError: list expected at most 1 argument, got 2
